analyzers/tests/SonarAnalyzer.UnitTest/Rules/ExpectedExceptionAttributeShouldNotBeUsedTest.cs

```diff
@@ -19,9 +19,10 @@
  */
 
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using SonarAnalyzer.Rules.CSharp;
 using SonarAnalyzer.UnitTest.MetadataReferences;
 using SonarAnalyzer.UnitTest.TestFramework;
+using CS = SonarAnalyzer.Rules.CSharp;
+using VB = SonarAnalyzer.Rules.VisualBasic;
 
 namespace SonarAnalyzer.UnitTest.Rules
```

Comment:
NITPICK: According to our [coding style](https://github.com/SonarSource/sonar-dotnet/blob/master/docs/coding-style.md#naming-conventions) the test methods should have _CS and _VB suffixes and not prefixes. Please rename the test methods.